source/CoolUtil.hx

```diff
@@ -128,4 +128,75 @@ class CoolUtil
 		FlxG.openURL(site);
 		#end
 	}
+	
+	
+	public function getProperty(variable:String) {
+		var killMe:Array<String> = variable.split('.');
+		if(killMe.length > 1) {
+			var coverMeInPiss:Dynamic = null;
```

Comment:
What does this variable even do other than have a weird name

Reply:
-- those variables are in the engine right now lol